osbs/build/build_response.py

```diff
@@ -119,6 +119,22 @@ def get_logs(self, decode_logs=True):
         else:
             return logs
 
+    def get_error_message(self):
+        """
+        Return an error message based on atomic-reactor's metadata
+        """
+        try:
+            str_metadata = graceful_chain_get(self.get_annotations_or_labels(), "plugins-metadata")
+            metadata_dict = json.loads(str_metadata)
+            plugin, error_message = list(metadata_dict['errors'].items())[0]
```

Comment:
It's reasonable I think. The first error changed the build from succeeding to failing.